src/NServiceBus.AcceptanceTests/EndpointTemplates/Requires.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    static class Requires
```

Comment:
nope, these are just the other "transport descriptions" which can be used in the `Repeat` statement.
e.g. `Repeat(x => x.For<AllTransportsWithoutNativeDeferral>())` would be the current corresponding configuration to `NativeDeferralSupport`.